src/contentmanager.cpp

```diff
@@ -63,7 +63,7 @@ QStringList ContentManager::getBookInfos(QString id, const QStringList &keys)
     if (nullptr == b){
         for(auto& key:keys) {
             (void) key;
-            values.append("");
+            values.insert(key, "");
         }
         return values;
     }
```

Comment:
It is not the same question.
I speaking about the api of the content manager. Your question is interesting and need to be answered but it would be the same, what ever the function returns when you want information about a specific book and the book is not found.